### DISCUSSION TITLE
Tab auto-unloading time adjustment due to tab inactivity

### DISCUSSION THREAD

**Discussion by GHNewbiee** [Floorp-Projects/Floorp#1073] [Feature Request]
Please consider to add the above-mentioned feature in Settings to make access easier and faster.

*Tia*

**Comment by CutterKnife** [comment]
Is this setting different?

<img width="665" alt="スクリーンショット 2024-04-04 23 48 52" src="https://github.com/Floorp-Projects/Floorp/assets/110194318/86e31677-7dd4-41c0-ac9c-ec003479f980">

**Reply by GHNewbiee** [reply]
@CutterKnife *Tal* I thought that terms are more or less fixed. Anyway, it's not a big deal as long as it's there.
